### PR TITLE
Update xtensor to 0.15.9

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Dunno what is going on with the windows build.

```
[ 40%] Building CXX object test/CMakeFiles/test_xtensor_julia.dir/test_jlarray.cpp.obj
test_jlarray.cpp
C:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\um\winnt.h(3122): error C2733: '_mm_prefetch': second C linkage of overloaded function not allowed
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\xmmintrin.h(346): note: see declaration of '_mm_prefetch'
C:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\um\winnt.h(3128): error C2733: '_m_prefetchw': second C linkage of overloaded function not allowed
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\intrin.h(610): note: see declaration of '_m_prefetchw'
C:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\um\winscard.h(56): error C2373: 'LPCVOID': redefinition; different type modifiers
C:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\shared\minwindef.h(174): note: see declaration of 'LPCVOID'
C:\projects\julia-build\include\julia\julia.h(788): warning C4146: unary minus operator applied to unsigned type, result still unsigned
C:\Users\appveyor\.julia\v0.5\CxxWrap\deps\usr\include\jlcxx/type_conversion.hpp(1076): warning C4800: 'int8_t': forcing value to bool 'true' or 'false' (performance warning)
```

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
@JohanMabille @wolfv have ever encountered that windows issue before?

**Comment by SylvainCorlay** [comment]
cc @barche - in case you have seen this issue before!

File: REQUIRE [hunk 1]
```diff
@@ -1,3 +1,3 @@
 julia 0.6
 BinDeps 0.4.2
-CxxWrap 0.5.0
+CxxWrap 0.5.0 0.6.0
```
Comment (barche) [inline REQUIRE:3]: This should be `CxxWrap 0.5 0.6-`, otherwise it will still get the incompatible CxxWrap 0.6

**Comment by barche** [comment]
Sorry, haven't seen the error before. Is it possible there is some binary incompatibility between the Conda xtensor binary and the Visual Studio used to compile Xtensor.jl?

**Comment by SylvainCorlay** [comment]
Yep I don't know what is going on. Xtensor is header-only...

Sylvain

File: REQUIRE (hunk 1 above)
Comment (SylvainCorlay) [reply REQUIRE:3]: (As per discussion with @barche on another thread) the `-` is not requires.

**Comment by SylvainCorlay** [comment]
Closing as superseeded by  #76 .

**Pull request closed without merging**